sdk/python/feast/entity.py

```diff
@@ -75,6 +77,11 @@ def __init__(
         self._created_timestamp: Optional[datetime] = None
         self._last_updated_timestamp: Optional[datetime] = None
 
+        stack = inspect.stack()
+        # Get two levels up from current, to ignore usage.py
+        previous_stack_frame = stack[2]
+        self.defined_in = previous_stack_frame.filename
```

Comment:
Should we abstract that in an helper function and also test it in a test file ?
Don't think we need to test each class but at least the helper function

What do you think ?

Reply:
A helper class probably makes sense since there's some state tracking being done. wdyt @achals ?

Reply:
> A helper class

yeah even better 👍

Reply:
Done